cpp/iedriver/BrowserFactory.cpp

```diff
@@ -1393,4 +1435,50 @@ bool BrowserFactory::IsEdgeMode() const {
   return this->edge_ie_mode_;
 }
 
+// delete a folder recursively
+int DeleteDirectory(const std::wstring &dir_name) {
+  WIN32_FIND_DATA file_info;      
+
+  std::wstring file_pattern = dir_name + L"\\*.*";
+  HANDLE hFile = ::FindFirstFileW(file_pattern.c_str(), &file_info);
+  if(hFile != INVALID_HANDLE_VALUE) {
+    do {
+      if(file_info.cFileName[0] == '.') continue;
+      std::wstring file_path = dir_name + L"\\" + file_info.cFileName;
+
+      if(file_info.dwFileAttributes & FILE_ATTRIBUTE_DIRECTORY) {
+        int iRC = DeleteDirectory(file_path);
+        if(iRC) return iRC;
+      } else {
+        if(::SetFileAttributesW(file_path.c_str(), FILE_ATTRIBUTE_NORMAL) == FALSE)
+          return ::GetLastError();
+
+        if(::DeleteFileW(file_path.c_str()) == FALSE)
+          return ::GetLastError();
+      }
+    } while(::FindNextFileW(hFile, &file_info) == TRUE);
+
+    ::FindClose(hFile);
+    DWORD dwError = ::GetLastError();
+    if(dwError != ERROR_NO_MORE_FILES) 
```

Comment:
Consistency and readability: Use braces, even around one-statement branches.

Reply:
Added braces for one statement branches.